### PR TITLE
Loc staging

### PR DESCRIPTION
**Pull request by DandiKandie**
adding information for team to use.

### CODE REVIEW

**Review by traplocz — APPROVED** [review]
**User Kandie**: added colors reference within **_public/css/colors.css_**
and google link code font reference within **_public/fonts/font.txt_** 
with commit code **75a60e4**.

**User Kandie**:  deleted a google front link and changed it to a css font value
within **_public/fonts/font.txt_**
with commit code **843c43d**.

**User Kandie**:  deleted a css font value, then put it back again and also added google font link codes.
within **_public/fonts/font.txt_**
with commit code **1cf1103**.

### Basically it seems like Kandie revised her work over and over and each revision shows as a commit. The end result should all merge together correctly (hopefully).

**User Ahrianna23** : added 6 different additions within **_public/images/icons/bag-icon.svg_** and here comment on her commit says: "Uploading Icons (Search, Profile, Bag, Logo, & Menu)"
with commit code **bf8261d**.
